mdjs.config.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
Can be removed I think since we now have a defaut mdjs.config.js generated in vite too

Reply:
Removed it, works indeed